### ISSUE TITLE
Conflict of interest?

### ISSUE THREAD

**Issue by KTrain5169** [usevital/notessential#101]
Raised by @TheKodeToad in the Prism Discord

Technically, Vital makes money (albeit only to devin) from Modrinth ad revenue, so it could come across as a CoI because we are "competing" against Essential (though not actually)

The minimum to address this might be removing the Vital modpack from the /alternatives page, but suggestions are open on how to figure this one out

**Comment by blryface** [comment]
oh god, issue 101 already?

I think we should keep Vital in some way, because it's the easiest way for people to switch from using Essential. We should figure out a way to keep them separate in some way.

**Comment by worldwidepixel** [comment]
Notessential was the driving force of Vital's creation, not the other way around.


There probably is one anyway, but we all just generally dislike the Essential mod

**Comment by blryface** [comment]
> oh god, issue 101 already?

unsurprising now that i know every commit needs to be tied to a pr or something

**Comment by worldwidepixel** [comment]
The only possible long-term theoretical solution is to move NotEssential back to being 'under blurry', or at least not under Vital

This (or similar) probably won't be done until it's actually Vital (pun intended) to do so.

**Comment by KTrain5169** [comment]
I don't think it'd make sense since Translator & Ward are part of NotEssential (technically), which complicates things. Would these be under 'blurry's ownership'? Would these return to being back under `KTrain5169/` or `WorldWidePixel/`?

Also, no NotEssential maintainers actually get a share of the Vital revenue, and even if so the amount of money would be so small split 6-way that it'd be almost nothing (probably). idk seems more complicated than first thought

**Comment by blryface** [comment]
I think giving me NotEssential 'ownership' back would be a pain both for management and for contributors, because I can't give better permissions on a user repo.
We should figure out a way to keep them "separate" while being sort of together

> Also, no NotEssential maintainers actually get a share of the Vital revenue, and even if so the amount of money would be so small split 6-way that it'd be almost nothing (probably). idk seems more complicated than first thought

Whoever tf thinks it's a money thing because I'm earning a cent per month from Vital can go explode

**Comment by KTrain5169** [comment]
> I think giving me NotEssential 'ownership' back would be a pain both for management and for contributors, because I can't give better permissions on a user repo. We should figure out a way to keep them "separate" while being sort of together
Unfortunately the best way would be splitting into 2 orgs again, which, idk doesn't sound like the best idea because if we do vital will be by itself, then taken back by devin, therefore being a repeat of the previous situation which imho doesn't make sense

> > Also, no NotEssential maintainers actually get a share of the Vital revenue, and even if so the amount of money would be so small split 6-way that it'd be almost nothing (probably). idk seems more complicated than first thought
> 
> Whoever tf thinks it's a money thing because I'm earning a cent per month from Vital can go explode

based

**Comment by blryface** [comment]
Even if we do it on a slighltly less "best" way I think we should be fine.
What about not "promoting" the pack on the website? We could change it to be a modpack that we made for whoever has enough patience to go trough searching a bit to find it, and have it not be *directly* tied to the website on both ends

**Comment by KTrain5169** [comment]
Not "promoting" the pack on the website is good imo, I can't gauge what impact it has on Vital and NotEssential but personally dehoisting it to the bottom at least makes more sense, people don't want an "ad" for it at the top if they go to the alternatives page, and people who don't want to install them can just scroll down I suppose?

Idea probably needs some more thinking

**Comment by blryface** [comment]
I think removing it completely would be fine. Vital is getting more talked about than Notessential, so it's not like it's gonna perform badly if we remove it.

**Comment by KTrain5169** [comment]
Sure I suppose

**Comment by worldwidepixel** [comment]
I agree that removing Vital from Notessential would not affect Vital's performance

**Comment by blryface** [comment]
I DID???

**Comment by blryface** [comment]
WHEN TF DID I

**Comment by blryface** [comment]
was the issue tied to the pr somehow :catstar:

**Comment by KTrain5169** [comment]
yes it was

**Comment by Thinkseal** [comment]
2 minutes ago

**Comment by blryface** [comment]
this is a very thread

**Comment by blryface** [comment]
wait I thought I did reopen this

**Comment by blryface** [comment]
# Issue reopened
We have internally discussed about issues in the ways we manage Notessential/Vital, so I'll be reopening this issue for further discussion if we need it.

**Comment by blryface** [comment]
Could we remove #12 for a while?
